Jenkinsfile.release

```diff
@@ -36,6 +36,10 @@ properties([
     ])
 ])
 
+if (params.VERSION == "") {
+    throw new Exception("Missing VERSION parameter!")
+}
+
```

Comment:
Is there a way to make it required in the properties above instead of doing the check here?

Reply:
Heh, yeah I was looking at that, and it led me to https://issues.jenkins-ci.org/browse/JENKINS-3509.

Reply:
Nice.. comment in the code here?